vllm/engine/llm_engine.py

```diff
@@ -1085,8 +1139,17 @@ def _advance_to_next_step(
             if seq_group.is_finished():
                 continue
 
-            seq_group.update_num_computed_tokens(
-                seq_group_metadata.token_chunk_size)
+            if seq_group_metadata.is_prompt:
+                if self.scheduler_config.is_multi_step_chunked_prefill:
+                    # Prompts are scheduled in multi-step only when
+                    # chunking is enabled. These prompts turn into
+                    # decodes after the very first step. Therefore,
+                    # we skip the updated to the num_computed_tokens
+                    # here.
+                    pass
```

Comment:
This may introduce a potential bug in the future when prompt chunking is supported. What issue will we see if we still update num_computed_tokens?

Reply:
If we update here, then the next call to `update_num_computed_tokens` will hit the assert https://github.com/vllm-project/vllm/blob/9e5ec35b1f8239453b1aaab28e7a02307db4ab1f/vllm/sequence.py#L270